JarrydWorland/2021_Capstone_GP2_Team
# Added Pokemon-style Narrative Display

### Changes Made

- Added three scripts (DialogueBehaviour.cs, Dialogue.cs & DialogueTrigger.cs)
- Added DialogueDisplay prefab to Canvas in Gameplay scene

### How to test

- Open up Gameplay scene and navigate past MenuNarrative to see DialogueDisplay in action
